migrate-core/src/lib.rs

```diff
@@ -37,39 +37,38 @@ use std::fmt;
 use tracing::{info, info_span, instrument};
 use tracing_futures::Instrument;
 
-/// Contains the behavior of a single migration that may be applied or reversed
+/// Contains behavior of a single migration that may be applied or reversed
 /// using [`Migration::up()`] and [`Migration::down()`] methods respectively.
 #[async_trait]
 pub trait Migration: Send + 'static {
-    /// Defines that type of the context that will be injected for the migration
-    /// to use during its execution.
+    /// Defines context type injected for migration during its execution.
     ///
-    /// The context will be created by [`MigrationCtxProvider`] and looked up by
+    /// Context will be created by [`MigrationCtxProvider`] and looked up by
     /// its [`type_id()`][std::any::Any::type_id].
     type Ctx: Send + 'static;
 
-    /// Run the forward migration logic. The given [`Migration::Ctx`] should
-    /// be used to perform the execution. The context should take care to
-    /// to commit the changes to the target migration object (e.g. a database)
-    /// or just collect the diagnostic info about the planned operations
+    /// Run forward migration logic. To perform the execution given 
+    /// [`Migration::Ctx`] should be used. The context should commit 
+    /// the changes to the target migration object (e.g. a database)
+    /// or just collect the diagnostic info about planned operations
     /// according to the [`MigrationRunMode`].
     ///
-    /// The migration is safe to assume that migrations that precede it were
-    /// already applied and it may observe the changes made by them.
+    /// For a migration it is safe to assume that preceding migrations were
+    /// already applied and it may observe changes they made.
     async fn up(&mut self, ctx: &mut Self::Ctx) -> Result<(), DynError>;
 
-    /// Similar to [`Migration::up()`], but applies the migration logic in reverse
+    /// Similar to [`Migration::up()`], but applies migration logic in reverse
     /// direction. It may safely assume that this same [`Migration::up()`]
-    /// method was run and it may observe the changes made by the forward
+    /// method was run and it may observe changes made by forward
     /// migration logic.
     ///
-    /// This method should cancel the changes made by the forward migration logic
-    /// and basically rollback the state of the migration object to the state
+    /// This method should cancel changes made by forward migration logic
+    /// and basically rollback the state of migration object to the state
     /// it was before [`Migration::up()`] was called.
     async fn down(&mut self, ctx: &mut Self::Ctx) -> Result<(), DynError>;
 }
 
-/// Bbuilder for [`Plan`] to allow its convenient configuration
+/// Builder for [`Plan`] to allow it's convenient configuration
```

Comment:
```suggestion
/// Builder for [`Plan`] to allow its convenient configuration
```
It's is short for `it is`, but here `its` is [correct](https://www.faa.gov/about/initiatives/plain_language/articles/its/)

Reply:
aha, ok, I missed some context there